src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -670,7 +670,12 @@
 helps['storage copy'] = """
 type: command
 short-summary: Copy files or directories to or from Azure storage.
-long-summary: Copy command depends on Azcopy, which only works for 64-bit Operating System now. We will support 32-bit Operating System soon.
+long-summary: > 
+    Copy command depends on Azcopy, which will be upgraded to v10.3 soon to support 32-bit Operating System and
+    utilize new features.
+    
+    [COMING BREAKING CHANGE] With Azcopy v10.3, `*` character is no longer supported as a wildcard in URL, but new
```

Comment:
in URL means `--url`?

Reply:
No. --source and s--destination support url as parameter value.